docs/elements/overview.mdx

```diff
@@ -0,0 +1,68 @@
+---
+description: Clerk Elements is a library of unstyled, composable components that can be used to build completely custom UIs on top of Clerk's APIs, without having to manage the underlying logic.
+---
+
+# Clerk Elements (Beta)
+
+<Callout type="warning">
+Clerk Elements is currently in beta. While it is **not** yet recommended for production use, it would be much appreciated if you gave it a try.
+
+If you have any feedback, please reach out to [beta-elements@clerk.dev](mailto:beta-elements@clerk.dev) or head over to the [GitHub Discussion](#TODO).
+</Callout>
+
+Clerk Elements is a library of unstyled, composable components that can be used to build completely custom UIs on top of Clerk's APIs, without having to manage the underlying logic.
+
+## Getting started
+
+Clerk Elements currently only works with Next.js and [Clerk Core 2](https://clerk.com/changelog/2024-04-19). As it gets closer to a stable release, support for additional frameworks will be added. If your Next.js application is already using Clerk, make sure to [upgrade to Core 2](/docs/upgrade-guides/core-2/nextjs). If you're starting from scratch, follow the [Next.js quickstart](/docs/quickstarts/nextjs) before proceeding.
```

Comment:
Does it matter which version of Next.js?

Reply:
Yeah, one needs to use `^13.5.4 || ^14.0.3`

Reply:
This is already specified in our Next.js SDK, so I don't think we need to explicitly call it out here.

Reply:
Ah, okay, that makes sense